packages/ra-ui-materialui/src/field/ImageField.spec.js

```diff
@@ -28,14 +28,14 @@ describe('<ImageField />', () => {
             />
         );
 
-        const img = wrapper.find('img');
-        assert.equal(img.prop('src'), 'http://foo.com/bar.jpg');
-        assert.equal(img.prop('alt'), 'Hello world!');
-        assert.equal(img.prop('title'), 'Hello world!');
+        const img = getByRole('img');
```

Comment:
use either `getByAltText` or `getByTitle`: https://testing-library.com/docs/dom-testing-library/api-queries#byalttext

Reply:
That way, you can avoid the assertion on either the title or alt attribute

Reply:
Okay, thank you :)